package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "rlbotjavascriptexample",
+  "version": "1.0.0",
+  "description": "An example bot implemented in JavaScript",
+  "main": "main.js",
+  "license": "ISC",
```

Comment:
We use MIT for everything so far, I would prefer to stick with that as a standard for now.